homeassistant/components/mqtt/fan.py

```diff
@@ -235,7 +235,7 @@ def _setup_from_config(self, config):
             self._topic[CONF_OSCILLATION_STATE_TOPIC] is not None and SUPPORT_OSCILLATE
```

Comment:
Should we fix this too?

Reply:
Yeah definitely.

Reply:
Done https://github.com/home-assistant/home-assistant/pull/28680/commits/ce4f82bdbe81c3e5d7786131163900ce503cfb08